packages/integrations/scripts/buildBundles.ts

```diff
@@ -0,0 +1,22 @@
+import { spawnSync } from 'child_process';
+import { readdirSync } from 'fs';
+import { join } from 'path';
+
+function getIntegrations(): string[] {
+  // The index file is only there for the purposes of npm builds (for the CDN we create a separate bundle for each
+  // integration) and the flags file is just a helper for including or not including debug logging, whose contents gets
+  // incorporated into each of the individual integration bundles, so we can skip them both here.
+  return readdirSync(join(__dirname, '..', 'src')).filter(file => !file.endsWith('index.ts'));
```

Comment:
isn't the shell script using a `==` check instead of a substring check? Why did we do `endsWith` here?

Also nit: would be nice to extract these into vars so you know all the return values.

```ts
const fileNames = join(__dirname, '..', 'src');
const files = readdirSync(fileNames);
return files.filter(...);
```